kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -26,20 +26,32 @@
             {{form.stripe_public_key}}
         {% endif %}
 
-        {% if recurring_payment %}
-            <input type="hidden" id="stripe_source_setup" value="1">
-        {% endif %}
 
         <p class="legal-copy">
-            {% trans %}
+            {% trans terms_url=url('payment_terms') %}
```

Comment:
This may be a good time to remove the translation block from this legal copy as well (https://trello.com/c/X78QBZWn/55-bug-legal-copy-should-be-english-only)